packages/devtools_app/lib/src/shared/primitives/simple_items.dart

```diff
@@ -50,3 +50,26 @@ const String traceEventsFieldName = 'traceEvents';
 const closureName = '<closure>';
 
 const anonymousClosureName = '<anonymous closure>';
+
+const _memoryDocUrl =
+    'https://docs.flutter.dev/development/tools/devtools/memory';
+const _consoleDocUrl =
+    'https://docs.flutter.dev/development/tools/devtools/console';
+
+enum DocLinks {
+  chart(_memoryDocUrl, 'expandable-chart'),
+  profile(_memoryDocUrl, 'profile-memory-tab'),
+  diff(_memoryDocUrl, 'diff-snapshots-tab'),
+  trace(_memoryDocUrl, 'trace-instances-tab'),
+  console(_consoleDocUrl, null),
```

Comment:
remove the last comma and replace with semicolon so we don't have a single line with a semicolon below

Reply:
It is easier to reorganize items, when they all have the same sign at the end. 
We have single line with semicolon in other places of the app (search by `  ;`). It looks as good separator between items and fields/methods.

Reply:
I am open to fix all single semicolons in next PR if you still think it is right thing to do.

Reply:
In general having a comma and a semi-colon seems messy. I'm in favor of fixing all uses of this in a follow-up PR.  Example from flutter: https://dart.dev/language/enum#declaring-enhanced-enums.